### DISCUSSION TITLE
Default Login Credentials for StartOS

### DISCUSSION THREAD

**Discussion by cowcumber** [Start9Labs/start-os#2800] [Q&A] [answered]
Hi everyone,  
I've installed StartOS (aarch64) in a virtual machine to test it out. After completing the initial setup and installation via the web interface, I encountered an issue. When I open the virtual machine, it displays the classic Linux login prompt:

```
Debian GNU/Linux 12 ancient-sashes ttyAMA0

ancient-sashes login:
```

I've searched the documentation for the default username and password but couldn’t find any information. Does anyone know the login credentials or how to access the system?  

Thanks in advance for your help!

**Comment by k0gen** [comment, marked as answer]
Hi there!

It looks like you're trying to access your StartOS system via the terminal login prompt. Here's what you need to know:

1. **Default Username**: The default user for SSH login is `start9`. This is the username you should use when logging in.

2. **Password Login**: By default, password login is disabled for security reasons. Instead, StartOS uses SSH key authentication for logging in. The SSH key is provided through the StartOS UI (System > SSH). If you haven't set up an SSH key yet, you'll need to do so via the StartOS web interface.

3. **Development Builds**: If you're using a [development build](https://github.com/Start9Labs/start-os/actions/workflows/startos-iso.yaml) of StartOS, password login is enabled as an exception. In this case, the username is still `start9`, and the password is `embassy`.

Good Luck!
